tests/unit/command/test_experiments.py

```diff
@@ -98,11 +95,11 @@ def test_experiments_run(dvc, scm, mocker, args, reset):
         "jobs": None,
         "tmp_dir": False,
         "checkpoint_resume": None,
-        "reset": reset,
+        "reset": False,
```

Comment:
Is a `"reset": True` case needed?

Reply:
The command unit tests are mainly for making sure the CLI parsing works. The actual reset=true/false functionality is tested in `tests/func/experiments`.